s3/pkg/service.go

```diff
@@ -48,6 +48,23 @@ var SupportedClasses []pb.StorageClass
 
 type s3Service struct{}
 
+func getTierFromName(backendType, className string) (int32, S3Error) {
+	v, ok := Ext2IntTierMap[backendType]
+	if !ok {
+		log.Logf("get tier of storage class[%s] for backend type of [%s] failed.\n", className, backendType)
+		return 0, InternalError
+	}
+
+	v2, ok := (*v)[className]
+	if !ok {
+		log.Logf("get tier of storage class[%s] for backend type of [%s] failed.\n", className, backendType)
+		return 0, InternalError
```

Comment:
i think we need to distinguish this error from the previous one...did we not find the backendType OR did we not find the storageClass in the backendType (we found the backendType) ?

Reply:
Done